packages/@vue/cli-service/package.json

```diff
@@ -46,7 +46,7 @@
     "cli-highlight": "^2.1.4",
     "clipboardy": "^2.3.0",
     "cliui": "^6.0.0",
-    "copy-webpack-plugin": "^5.1.1",
+    "copy-webpack-plugin-v5": "^5.1.2",
```

Comment:
Maybe an idea to make this a fixed version, because of security.
But I don't know.